.github/workflows/deploy_storybook.yaml

```diff
@@ -0,0 +1,61 @@
+name: Deploy Storybook
+
+on:
+  pull_request:
+    types:
+      - closed # Trigger when the PR is closed
+    branches:
+      - 'main' # Target branch is main
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+    working-directory: apps/nextjs-website
+
+# These permissions are needed to interact with GitHub's OIDC Token endpoint.
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+
+  build:
+    name: Build and Deploy Storybook
+    runs-on: ubuntu-latest
+    
+    concurrency:
+      group: ${{ github.workflow }}
+      cancel-in-progress: false
+      
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+        
+      - name: Setup Node
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: "18.17"
```

Comment:
**question(non-blocking)**: why hardcode this version?

I think you can use the `node-version-file` parameter, or, even better, re-use a workflow already present in the repository.
Take a look [here](https://github.com/pagopa/b2b-portals/blob/main/.github/actions/setup-node/action.yaml)

Reply:
"uses: ./.github/actions/setup-node" in new commit